spec/services/match_service_spec.rb

```diff
@@ -67,7 +67,30 @@
       end
     end
 
-    # TODO: matches right teams for !powers of 2
+    [
```

Comment:
I don't like these test types, because we will always miss some number and run into an untested case which will cause errors... please create a ticket tracking the progress in finding a better approach for this.

Reply:
https://jiratinf16b4.it.dh-karlsruhe.de:8443/browse/TURNIERE-161